app/instances/db.py

```diff
@@ -0,0 +1,17 @@
+from app.server import server
+from flask_sqlalchemy import SQLAlchemy
+import redis
+import config
+
+# SQL
+db_config = config.db_config
+server.config['SQLALCHEMY_DATABASE_URI'] = f"mysql+mysqlconnector://{db_config['user']}:{db_config['password']}@{db_config['host']}:{db_config['port']}/{db_config['database']}"
+server.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+db = SQLAlchemy(server)
+
+# Redis
+r = redis.StrictRedis(**config.redis_config)
```

Comment:
This needs a more meaningful name.

Reply:
In all python-redis things on the web, they all use `r` which makes me thing it's like `app` for the flask object. so not sure